android/src/main/java/io/invertase/googlemobileads/ReactNativeAppModule.java

```diff
@@ -55,6 +57,17 @@ public void initializeApp(ReadableMap options, ReadableMap appConfig, Promise pr
     promise.resolve(options);
   }
 
+  @ReactMethod(isBlockingSynchronousMethod = true)
+  public int getCurrentOrientationAnchoredAdaptiveBannerHeight() {
```

Comment:
We already have a similar static method: https://github.com/invertase/react-native-google-mobile-ads/blob/main/android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsCommon.java#L43